plugins/modules/openssl_signature_info.py

```diff
@@ -0,0 +1,353 @@
+    - This module allows one to verify a signature for a file via a certificate.
+    - The module can use the cryptography Python library, or the pyOpenSSL Python
+      library. By default, it tries to detect which one is available. This can be
+      overridden with the I(select_crypto_backend) option. Please note that the PyOpenSSL backend
+      was deprecated in Ansible 2.9 and will be removed in community.crypto 2.0.0.
+requirements:
+    - Either cryptography >= 1.4 (some key types require newer versions)
+    - Or pyOpenSSL >= 0.11 (Ed25519 and Ed448 keys are not supported with this backend)
+author:
+    - Patrick Pichler (@aveexy)
+    - Markus Teufelberger (@MarkusTeufelberger)
+options:
+    path:
+        description:
+            - The signed file to verify.
+            - This file will only be read and not modified.
+        type: path
+        required: true
+    certificate_path:
+        description:
+            - The path to the certificate used to verify the signature.
+            - Either I(certificate_path) or I(certificate_content) must be specified, but not both.
+        type: path
+    certificate_content:
+        description:
+            - The content of the certificate used to verify the signature.
+            - Either I(certificate_path) or I(certificate_content) must be specified, but not both.
+        type: str
+    signature:
+        description: Base64 encoded signature.
+        type: str
+        required: true
+    select_crypto_backend:
+        description:
+            - Determines which crypto backend to use.
+            - The default choice is C(auto), which tries to use C(cryptography) if available, and falls back to C(pyopenssl).
+            - If set to C(pyopenssl), will try to use the L(pyOpenSSL,https://pypi.org/project/pyOpenSSL/) library.
+            - If set to C(cryptography), will try to use the L(cryptography,https://cryptography.io/) library.
+        type: str
+        default: auto
+        choices: [ auto, cryptography, pyopenssl ]
+notes:
+    - |
+      When using the C(cryptography) backend, the following key types require at least the following C(cryptography) version:
+      RSA keys: C(cryptography) >= 1.4
+      DSA and ECDSA keys: C(cryptography) >= 1.5
+      ed448 and ed25519 keys: C(cryptography) >= 2.6
+seealso:
+    - module: openssl_signature
+    - module: x509_certificate
+'''
+
+EXAMPLES = r'''
+- name: Sign example file
+  community.crypto.openssl_signature:
+    privatekey_path: private.key
+    path: /tmp/example_file
+  register: sig
+
+- name: Verify signature of example file
+  community.crypto.openssl_signature_info:
+    certificate_path: cert.pem
+    path: /tmp/example_file
+    signature: "{{ sig.signature }}"
+  register: verify
+
+- name: Make sure the signature is valid
+  assert:
+    that:
+      - verify.valid
+'''
+
+RETURN = r'''
+valid:
+    description: True means the signature was valid for the given file, False means it wasn't.
```

Comment:
```suggestion
    description: C(true) means the signature was valid for the given file, C(false) means it wasn't.
```
Jinja2 booleans are lower case.